pkg/internal/itest/http_fetch_test.go

```diff
@@ -0,0 +1,574 @@
+package itest
```

Comment:
this is.... awesome. 👍 💯 🥇 🙏 👏  + any other emojis the kids are into.

Reply:
and I did actually read most of it. It's just great we can spin these up and test so succinctly